### PR TITLE
SRCH-2979 reduce "size" of LowCtrQuery aggregation

### PR DESCRIPTION
**Pull request by MothOnMars**
## Summary
This PR reduces the `size` of the `LowCtrQuery` aggregation to resolve the following deprecation warning:

> This aggregation creates too many buckets (10001) and will throw an error in future versions. You should update the [search.max_buckets] cluster setting or use the [composite] aggregation to paginate all buckets in multiple requests.

The current `size` setting of 100,000 buckets is much larger than we need for that query, which is used to display analytics data in the GUI, as well as for the LowCtrQuery watchers. I'm dialing it down to 5000, which should be more than enough for our existing use cases.
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review:

#### Functionality Checks

- [x] Code is functional.

- [x] Automated checks pass, if applicable. If Code Climate checks do not pass, explain reason for failures:

- [x] If your changes will be tested manually, you have run `bundle update` and committed your changes to Gemfile.lock.
 
- [x] You have merged the latest changes from the target branch (usually `master` or `main`) into your branch.
 
- [x] If your target branch is NOT `master` or `main`, specify the reason:
 
- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release #.#.#** matching the release number
 
- [x] You have squashed your commits into a single commit (exception: your PR includes commits with formatting-only changes, such as required by Rubocop)

- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket
 
#### Process Checks

- [x] You have specified an "Assignee", and if necessary, additional reviewers

### CODE REVIEW

File: spec/models/logstash_queries/watcher_low_ctr_query_spec.rb [hunk 1]
```diff
@@ -28,50 +28,50 @@
           "filter": [
             {
               "term": {
-                "params.affiliate": 'affiliate_name'
```
Comment (MothOnMars) [inline spec/models/logstash_queries/watcher_low_ctr_query_spec.rb:31]: The logstash query specs are an exception to our normal Rubocop rule that enforces single over double quotes. Double-quotes are preferable in these specs as they allow us to copy and paste directly into Kibana for development and troubleshooting.

Comment (jamesmadhoc) [reply spec/models/logstash_queries/watcher_low_ctr_query_spec.rb:31]: Didn't remember that ruby allows underscores in number literals -- can completely understand that that wouldn't play nicely with JSON.  Didn't know the latter dislikes single quotes, either.

**Review by jamesmadhoc — APPROVED** [review]
Curious about how you arrived at 5000, a 20X reduction in that size.

Other than that, did a little find/greppery and seems you found all the locations for the change, so...

LGTM

**Comment by MothOnMars** [comment]
Good q - the limit needed to be below 10k, as that's the default `max_buckets` in ES 7. However, the limit actually needs to be below the `max_buckets`, as ES pulls additional results form each shard, and then processes the combined result set. See:
- https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-terms-aggregation.html#search-aggregations-bucket-terms-aggregation-shard-size
- https://stackoverflow.com/a/57464229

5k is still somewhat arbitrary, but it gets us well below the threshold, and is still returning more data than we actually display to the user.

**Pull request merged**